source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -335,9 +335,25 @@
         "type": "string"
       }
     },
+    "babel": {
+      "type": "object",
+      "properties": {
+        "options": {
+          "type": "object",
+          "description": "Options given to @babel/preset-env. With this options the output type of babel can be defined. For details see here: <https://babeljs.io/docs/en/babel-preset-env#options>. They can be overridden per target."
+        },
+        "plugins": {
+          "type": "object",
+          "description": "List of additional babel plugins to enable, eg TC39 proposals; note that some proposals will work out of the box and some will require upgrades to QxCompiler before they work properly - your mileage may vary.  Each key in this object is the NPM name of the plugin, and the value is either true (enabled), false (ignore), or an object which is options to pass to the preset",
+          "additionalProperties": {
+            "type": [ "boolean", "object" ]
+          }
+        }
+      }
+    },
     "babelOptions": {
       "type": "object",
-      "description": "Options given to @babel/preset-env. With this options the output type of babel can be defined. For details see here: <https://babeljs.io/docs/en/babel-preset-env#options>. They can be overridden per target."
+      "description": "** DEPRECATED - See babel.options instead"
```

Comment:
This is only valid for the top level babelOptions element. So you should copy the old description to the target/babelOptions tag.

If we update the schema to https://json-schema.org/draft/2019-09/schema we could use the deprecated keyword.

See https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.3